src/enterprise/enterprise.service.ts

```diff
@@ -29,7 +29,7 @@ export class EnterpriseService {
     return this.enterprisesRepository.find();
   }
 
-  findOne(id: string): Promise<Enterprise> {
+  findOne(id: any): Promise<Enterprise> {
```

Comment:
You should avoid using any in TypeScript, is this very needed ?

Reply:
Done